### ISSUE TITLE
PreviewImage node (bug & suggestion)

### ISSUE THREAD

**Issue by DadoIrie** [11cafe/comfyui-workspace-manager#69]
![image](https://github.com/11cafe/comfyui-workspace-manager/assets/61109153/cfe8c1d0-3b5a-42bb-af19-14f5d9b36ff5)
![image](https://github.com/11cafe/comfyui-workspace-manager/assets/61109153/feefc509-91f5-4c3a-9d9f-7c540d59ffef)

preview image node should not be saved in the gallery and currently it seems like there is no distinction being made between save and preview node - which leads to the issue shown in the screenshot
might I suggest placing images in the gallery only when saving them with comfyUI standard node or any custom node - keep in mind also that images can be saved somewhere else other than in the output folder

edit: haven't checked if another issue was opened addressing this and after seeing the screenshots in the matrix channel I assume that this issue is known too
edit2: small off-topic additional suggestion: delete image within the gallery would be nice too :)

edit3:

![image](https://github.com/11cafe/comfyui-workspace-manager/assets/61109153/56d9d168-07be-4c7b-8451-84d991b33c6a)
when deleting the saved image in the output folder 

edit4
![image](https://github.com/11cafe/comfyui-workspace-manager/assets/61109153/7d1e7c61-67a8-4f5b-bb20-a637cd4f746d)
background panel does not grow with the amount of images

**Comment by NewEpoch2020** [comment]
I have also noticed this problem, and I think the developer may be able to add a delete button for each picture, so that non-existent pictures can be manually cleaned up. 

In addition, because the temp folder will be automatically deleted after the comfyui client is closed, so I  think you can also automatically clean up the images belonging to the temp path before starting the plug-in every time.

![image](https://github.com/11cafe/comfyui-workspace-manager/assets/75068628/ec5a2776-b6ab-4e16-97e8-4e40516efb4c)

**Comment by Weixuanf** [comment]
hi these bugs are now fixed with https://github.com/11cafe/comfyui-workspace-manager/pull/70: 
- images overflow gallery when image exceed the modal size
- preview images are not saved to gallery anymore, only the ones in your output folder will be saved to gallery
- delete button added to remove images from gallery
<img width="624" alt="Screenshot 2024-01-04 at 3 58 40 PM" src="https://github.com/11cafe/comfyui-workspace-manager/assets/18367033/b9702b32-f60d-4454-9c71-b76d1dcce602">
